Neos.FluidAdaptor/Tests/Unit/ViewHelpers/Validation/ResultsViewHelperTest.php

```diff
@@ -55,9 +55,9 @@ public function renderAddsValidationResultsToTemplateVariableContainer()
     {
         $mockValidationResults = $this->getMockBuilder(\Neos\Error\Messages\Result::class)->getMock();
         $this->request->expects(self::atLeastOnce())->method('getInternalArgument')->with('__submittedArgumentValidationResults')->will(self::returnValue($mockValidationResults));
-        $this->templateVariableContainer->expects(self::at(0))->method('add')->with('validationResults', $mockValidationResults);
+        $this->templateVariableContainer->method('add')->with('validationResults', $mockValidationResults);
```

Comment:
Does this not need an `once()` assertion? Same for the other cases below

Reply:
Good catch, thanks!